src/screens/IndexScreen.js

```diff
@@ -120,9 +133,14 @@ export const IndexScreen = ({ navigation, route }) => {
       [QUERY_TYPES.NEWS_ITEMS]: showNewsFilter
     }[query];
```

Comment:
isn't the new filter missing here?

Reply:
now we have an extra query for addresses and a separate branch has been created for the refactor as we discussed.
https://github.com/smart-village-solutions/smart-village-app-app/commit/88d315d19937921c6729c08bef7e3d3ba2e03260